client/components/card-heading/index.jsx

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { includes } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+
+const validTypeSizes = [ 54, 47, 36, 32, 24, 21, 16, 14, 11 ];
+
+function CardHeading( { tagName = 'h1', size = 21, children } ) {
+	const classNameObject = {};
+	classNameObject[ 'card-heading-' + size ] = includes( validTypeSizes, size );
+	const classes = classNames( 'card-heading', classNameObject );
+	return React.createElement( tagName, { className: classes }, preventWidows( children, 2 ) );
+}
+
+CardHeading.propTypes = {
+	tagName: PropTypes.string,
```

Comment:
I know this is already merged, but what about using `PropTypes.oneOf( [ 'h1', 'h2', 'h3', 'h4', 'h4', 'h6' ] )` to make sure the title is using semantic tags?

Reply:
Great idea. Fancy creating a PR?